site/role/manifests/ssh.pp

```diff
@@ -0,0 +1,71 @@
+class role::ssh (
```

Comment:
1. Again, I don't understand how `ssh` can be a `role`.
2. Please fix tabs to spaces and set your editor accordingly. @alvagante Do you think we should introduce an `.editorconfig` file?

Reply:
Can you add upstream and we'll rebase? We're a bit out of sync with upstream at the moment. I'd suggest something like:
```# Editor configuration, see http://editorconfig.org
root = true

[*]
charset = utf-8
end_of_line = lf
indent_style = space
indent_size = 2
insert_final_newline = true
trim_trailing_whitespace = true

[*.md]
max_line_length = off
trim_trailing_whitespace = false```

Reply:
Added, will be merged to psick control repo soon.
Note that current psick has already the hieradata moved to a separated module.
So when you merge you'll have to manage hiera.yaml and Puppetfile as needed (can help with that if needed)

Reply:
Yes, I know we spoke about it, but I'd still like to set a meeting to do it. @shapi78 Please install editorconfig plugin in your editor of choice. :)